File: problemas/iniciante/1043/1043.py
A, B, C = [float(x) for x in input().strip().split(' ')]

if(A < B + C and B < A + C and C < A + B):
    print(f"Perimetro = {(A + B + C):.1f}")
else:
    print(f"Area = {((A + B)/2 * C):.1f}")

File: problemas/iniciante/1043/1043.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

let [A, B, C] = lines.shift().trim().split(' ').map((x) => parseFloat(x));

if (A < B + C && B < A + C && C < A + B) {
    console.log(`Perimetro = ${(A + B + C).toFixed(1)}`);
} else {
    console.log(`Area = ${((A + B) / 2 * C).toFixed(1)}`);
}